Feature/bigquery connector

rajeshhegde-datalicious opened the pull request:
@kmohaia-datalicious  https://datalicious.atlassian.net/browse/OH-730

Note: I have also pulled latest code from AirBnb master to data-licious/develop.

rajeshhegde-datalicious committed:
BigQuery connector base classes
	new file:   superset/connectors/bigquery/__init__.py
	new file:   superset/connectors/bigquery/models.py
	new file:   superset/connectors/bigquery/views.py

rajeshhegde-datalicious committed:
bigquery test base class
	new file:   tests/bigquery_tests.py

rajeshhegde-datalicious committed:
changes required for installing locally
	modified:   setup.py

rajeshhegde-datalicious committed:
create BigQuery Table view

	modified:   superset/config.py
	modified:   superset/connectors/bigquery/models.py
	modified:   superset/connectors/bigquery/views.py
	modified:   superset/views/core.py

rajeshhegde-datalicious committed:
bigquery table migration code
	modified:   alembic.ini
	modified:   superset/connectors/bigquery/models.py
	new file:   superset/migrations/versions/6f23fece6036_create_bigquery_tables.py

rajeshhegde-datalicious committed:
bigquery_table column changes
	modified:   superset/connectors/bigquery/models.py
	modified:   superset/connectors/bigquery/views.py
	modified:   superset/migrations/versions/6f23fece6036_create_bigquery_tables.py

rajeshhegde-datalicious committed:
bigquery table changes
	modified:   superset/connectors/bigquery/models.py
	modified:   superset/connectors/bigquery/views.py

rajeshhegde-datalicious committed:
get_perm method added
	modified:   superset/connectors/bigquery/models.py
	modified:   superset/utils.py

rajeshhegde-datalicious committed:
remove table_id from editing list
	modified:   superset/connectors/bigquery/models.py
	modified:   superset/connectors/bigquery/views.py

rajeshhegde-datalicious committed:
ability to add columns and metrics
	modified:   superset/connectors/bigquery/models.py
	modified:   superset/connectors/bigquery/views.py
	modified:   superset/migrations/versions/6f23fece6036_create_bigquery_tables.py
	modified:   superset/utils.py
	modified:   superset/views/core.py

kmohaia-datalicious reviewed (approved):
Thanks, I have reviewed the changes.

kmohaia-datalicious committed:
Merge branch 'develop' into feature/bigquery-connector

rajeshhegde-datalicious committed:
refactor

	modified:   superset/connectors/bigquery/models.py
	modified:   superset/migrations/versions/6f23fece6036_create_bigquery_tables.py

rajeshhegde-datalicious committed:
Merge branch 'feature/bigquery-connector' of github.com:data-licious/superset into feature/bigquery-connector

rajeshhegde-datalicious committed:
columns update
	modified:   superset/migrations/versions/6f23fece6036_create_bigquery_tables.py

rajeshhegde-datalicious committed:
config changes
	modified:   superset/config.py

rajeshhegde-datalicious committed:
get bigquery table name
	modified:   superset/utils.py

rajeshhegde-datalicious committed:
fetch columns from bigquery table and generate metric
	modified:   superset/connectors/bigquery/models.py
	modified:   superset/connectors/bigquery/views.py
	modified:   superset/views/core.py

rajeshhegde-datalicious committed:
refactor
 On branch feature/bigquery-connector
	modified:   superset/connectors/bigquery/models.py
	modified:   superset/connectors/bigquery/views.py

rajeshhegde-datalicious committed:
column changes
	modified:   superset/migrations/versions/6f23fece6036_create_bigquery_tables.py

rajeshhegde-datalicious committed:
revert column changes
	modified:   superset/migrations/versions/6f23fece6036_create_bigquery_tables.py

rajeshhegde-datalicious committed:
refactor
 On branch feature/bigquery-connector
	modified:   superset/connectors/bigquery/models.py
	modified:   superset/connectors/bigquery/views.py

rajeshhegde-datalicious committed:
run sql query from sql editor
 On branch feature/bigquery-connector
	new file:   superset/assets/javascripts/BigQuerySqlLab/TODO.md
	new file:   superset/assets/javascripts/BigQuerySqlLab/actions.js
	new file:   superset/assets/javascripts/BigQuerySqlLab/components/AceEditorWrapper.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/components/Alerts.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/components/App.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/components/ColumnElement.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/components/CopyQueryTabUrl.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/components/DataPreviewModal.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/components/HighlightedSql.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/components/Link.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/components/QueryAutoRefresh.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/components/QueryHistory.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/components/QuerySearch.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/components/QueryTable.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/components/ResultSet.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/components/RunQueryActionButton.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/components/SouthPane.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/components/SqlEditor.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/components/SqlEditorLeftBar.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/components/TabbedSqlEditors.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/components/TableElement.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/components/VisualizeModal.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/constants.js
	new file:   superset/assets/javascripts/BigQuerySqlLab/index.jsx
	new file:   superset/assets/javascripts/BigQuerySqlLab/main.css
	new file:   superset/assets/javascripts/BigQuerySqlLab/reducers.js
	modified:   superset/assets/webpack.config.js
	modified:   superset/connectors/bigquery/views.py
	new file:   superset/templates/superset/bigquery_sqllab.html
	modified:   superset/views/core.py

rajeshhegde-datalicious committed:
get column based on query result
 On branch feature/bigquery-connector
	modified:   superset/views/core.py

rajeshhegde-datalicious committed:
hide left side bar
 On branch feature/bigquery-connector
	modified:   superset/assets/javascripts/BigQuerySqlLab/components/TabbedSqlEditors.jsx

rajeshhegde-datalicious committed:
query builder function definition
 On branch feature/bigquery-connector
	modified:   superset/connectors/bigquery/models.py

rajeshhegde-datalicious committed:
query builder for bigquery
 On branch feature/bigquery-connector
	modified:   superset/connectors/bigquery/models.py
	modified:   superset/connectors/bigquery/views.py
	modified:   superset/migrations/versions/6f23fece6036_create_bigquery_tables.py

rajeshhegde-datalicious committed:
bigquery query builder bug fixes
 On branch feature/bigquery-connector
	modified:   superset/connectors/bigquery/models.py
	modified:   superset/connectors/bigquery/views.py
	modified:   superset/migrations/versions/6f23fece6036_create_bigquery_tables.py

rajeshhegde-datalicious committed:
bug fixes
 On branch feature/bigquery-connector
	modified:   superset/connectors/bigquery/models.py

rajeshhegde-datalicious committed:
add column
 On branch feature/bigquery-connector
	modified:   superset/assets/webpack.config.js
	modified:   superset/connectors/bigquery/models.py
	modified:   superset/connectors/bigquery/views.py
	modified:   superset/migrations/versions/6f23fece6036_create_bigquery_tables.py

rajeshhegde-datalicious committed:
bug fix
 On branch feature/bigquery-connector
	modified:   superset/connectors/bigquery/models.py

rajeshhegde-datalicious committed:
fix version number
 On branch feature/bigquery-connector
	modified:   setup.py
